examples/pessimistic_lock/src/aggregate.rs

```diff
@@ -0,0 +1,68 @@
+use serde::{Deserialize, Serialize};
```

Comment:
I think you can include another example, so you can avoid rewriting this aggregate stuff

Reply:
Eh all other examples have their own aggregate, I just went with the slimmest I could think of (avoiding projectors and policies).

Reply:
Done, recycled the aggregate from another example.